src/hypervisor_bootloader/src/main.rs

```diff
@@ -359,7 +477,6 @@ fn load_hypervisor() -> usize {
 
     for index in 0..elf_header.get_num_of_program_header_entries() {
         if let Some(info) = elf_header.get_segment_info(index, program_header_pool) {
-            pr_debug!("{:#X?}", info);
```

Comment:
I think it would be better if this PR only included changes for u-boot support.

Reply:
Reverted this deletion